shared/packages/worker/src/worker/workers/genericWorker/expectationHandlers/lib/coreApi.ts

```diff
@@ -77,3 +78,28 @@ export type LoudnessScanResultForStream =
 			 * */
 			balanceDifference?: number
 	  }
+
+export enum CompressionType {
+	Unknown = 'unknown',
+	Intra = 'intra',
+	FixedDistance = 'fixed_distance',
+	VariableDistance = 'variable_distance',
+}
```

Comment:
I could do with descriptions of each type here.
Especially `Intra`, does that mean that the file _only_ contains intra frames?